package.json

```diff
@@ -5,17 +5,14 @@
   "description": "The BRP application",
   "scripts": {
     "start": "node .",
-    "start:dev":
-      "./node_modules/parallelshell/index.js 'DEBUG=* ./node_modules/nodemon/bin/nodemon.js .' 'npm run watch:scss'",
+    "start:dev": "./node_modules/parallelshell/index.js 'DEBUG=* ./node_modules/nodemon/bin/nodemon.js .' 'npm run watch:scss'",
     "watch:scss": "./node_modules/nodemon/bin/nodemon.js -e scss -x 'npm run sass'",
     "test": "NODE_ENV=test mocha",
     "lint": "./node_modules/eslint/bin/eslint.js .",
     "style": "./node_modules/jscs/bin/jscs **/*.js --config=./.jscsrc.json",
-    "quality":
-      "./node_modules/plato/bin/plato -r -x 'node_modules|reports|test' -d reports/plato .",
+    "quality": "./node_modules/plato/bin/plato -r -x 'node_modules|reports|test' -d reports/plato .",
```

Comment:
Not sure what you mean... it's npm running code from a package.json - it's going to know about your dependencies since it defines them.

It's certainly never tripped me up anyway.